keps/sig-node/2837-pod-level-resource-spec/README.md

```diff
@@ -0,0 +1,1458 @@
+   </td>
+   <td>Infinity
+   </td>
+  </tr>
+  <tr>
+   <td>Unspecified
+   </td>
+   <td>Specified
+   </td>
+   <td>Specified
+   </td>
+   <td>Specified
+   </td>
+   <td>Effective Pod Limit
+   </td>
+   <td>Specified
+<p>
+Pod Limit
+   </td>
+   <td>Specified
+<p>
+Container Request
+   </td>
+   <td>Specified
+<p>
+Container Limit
+   </td>
+  </tr>
+  <tr>
+   <td>Unspecified
+   </td>
+   <td>Specified
+   </td>
+   <td>Specified
+   </td>
+   <td>Unspecified
+   </td>
+   <td>Effective Pod Limit
+   </td>
+   <td>Specified
+<p>
+Pod Limit
+   </td>
+   <td>Specified
+<p>
+Container Request
+   </td>
+   <td>Effective
+<p>
+Pod Limit
+   </td>
+  </tr>
+  <tr>
+   <td>Unspecified
+   </td>
+   <td>Specified
+   </td>
+   <td>Unspecified
+   </td>
+   <td>Specified
+   </td>
+   <td>Effective Pod Limit
+   </td>
+   <td>Specified
+<p>
+Pod Limit
+   </td>
+   <td>Effective container limit
+   </td>
+   <td>Specified Container Limit
+   </td>
+  </tr>
+  <tr>
+   <td>Unspecified
+   </td>
+   <td>Specified
+   </td>
+   <td>Unspecified
+   </td>
+   <td>Unspecified
+   </td>
+   <td>Effective Pod Limit
+   </td>
+   <td>Specified
+<p>
+Pod Limit
+   </td>
+   <td>Implementation based min
+   </td>
+   <td>Effective Pod Limit
+   </td>
+  </tr>
+  <tr>
+   <td>Unspecified
+   </td>
+   <td>Unspecified
+   </td>
+   <td>Specified
+   </td>
+   <td>Specified
+   </td>
+   <td>Aggregated container requests
+   </td>
+   <td>Aggregated container limits
+   </td>
+   <td>Specified
+<p>
+Container Request
+   </td>
+   <td>Specified
+<p>
+Container Limit
+   </td>
+  </tr>
+  <tr>
+   <td>Unspecified
+   </td>
+   <td>Unspecified
+   </td>
+   <td>Specified
+   </td>
+   <td>Unspecified
+   </td>
+   <td>Aggregated container requests
+   </td>
+   <td>Infinity
+   </td>
+   <td>Specified
+<p>
+Container Request
+   </td>
+   <td>Infinity
+   </td>
+  </tr>
+  <tr>
+   <td>Unspecified
+   </td>
+   <td>Unspecified
+   </td>
+   <td>Unspecified
+   </td>
+   <td>Specified
+   </td>
+   <td>Aggregate of effective container requests
+   </td>
+   <td>Aggregate of effective container limits
+   </td>
+   <td>Effective Container Limits
+   </td>
+   <td>Specified Container Limits
+   </td>
+  </tr>
+  <tr>
+   <td>Unspecified
+   </td>
+   <td>Unspecified
+   </td>
+   <td>Unspecified
+   </td>
+   <td>Unspecified
+   </td>
+   <td>Aggregate of effective containers requests 
+   </td>
+   <td>Infinity
+   </td>
+   <td>Implementation defined minimum
+   </td>
+   <td>Infinity
+   </td>
+  </tr>
+</table>
+<br>
+
+  Following validation logic needs to be added:
+
+  1. Only CPU and memory resources are supported in pod level resource specifications, hence invalidate pod-level resource specification for other resources like gpu, storage, etc. 
+  2. requests
+      * `pod.spec.resources.requests[memory/cpu] >= sum(pod.spec.containers[*].resources.requests[memory/cpu]) + sum(pod.spec.initContainers[*].resources.requests[memory/cpu])` is valid.
```

Comment:
as commented below, my understanding that this comparison is only executed if pod level requests are defined, correct? If not defined - there is nothing to compare with

Reply:
if the assumption is that we will populate pod level requests with the effective value - please clarify. I didn't find the mention of it anywhere